### PR TITLE
Add functionality for access Details URL endpoint

### PR DESCRIPTION
**Pull request by aitorsanz**

### CODE REVIEW

File: team/src/main/webapp/WEB-INF/jsp/details.jsp [hunk 1]
```diff
@@ -1,14 +1,18 @@
+<%@ page language="java" %>
```
Comment (juan-vg) [inline team/src/main/webapp/WEB-INF/jsp/details.jsp]: I think you should also remove these lines

**Comment by juan-vg** [comment]
Resolve the conflicts, please.

**Review by juan-vg — CHANGES_REQUESTED** [review]
Resolve the conflicts, please.

**Comment by aitorsanz** [comment]
Ok

**Review by juan-vg — APPROVED** [review]

**Pull request merged**